packages/docusaurus/src/client/exports/ErrorBoundary.tsx

```diff
@@ -33,17 +33,13 @@ export default class ErrorBoundary extends React.Component<Props, State> {
     const {error} = this.state;
 
     if (error) {
-      const fallback = this.props.fallback ?? DefaultFallback;
-      return fallback({
-        error,
-        tryAgain: () => this.setState({error: null}),
-      });
+      const Fallback = this.props.fallback ?? DefaultFallback;
```

Comment:
Right, this fix is in place because we had users who swizzled `@theme/Error` and used a `useState` in it, and that broke, because we were calling the component here (!)

I'm fine to change `?? DefaultFallback` to `?? (props) => <DefaultFallback {...props} />`.